### PR TITLE
Merge from master (July 4th, 2019)

### PR DESCRIPTION
**Pull request by RatanShreshtha**

### CODE REVIEW

File: _travis/upload_coverage.sh [hunk 1]
```diff
@@ -3,6 +3,6 @@
 set -exo pipefail
 
 if [[ -e .coverage ]]; then
-    python -m pip install codecov
-    codecov --env TRAVIS_OS_NAME,NOX_SESSION
+    python3.6 -m pip install codecov
+    python3.6 -m codecov --env TRAVIS_OS_NAME,NOX_SESSION
```
Comment (pquentin) [inline _travis/upload_coverage.sh]: This change is incorrect, we want to use "python" in order to also support the Python 2 builds

Comment (RatanShreshtha) [reply _travis/upload_coverage.sh]: Ohh, yeah I missed that

**Comment by pquentin** [comment]
You're also missing a change from "Also skip dotted FQDN tests on TRAVIS_INFRA=unknown"

**Comment by pquentin** [comment]
Sorry, as you said to me in private chat, the `Also skip dotted FQDN tests on TRAVIS_INFRA=unknown` was merged earlier, which is why it does not appear here. 👍

**Pull request closed without merging**